src/Http.php

```diff
@@ -213,6 +213,24 @@ public function post($path, $parameters)
         return $this->sendRequest($request);
     }
 
+    /**
+     * Retrieves http response for a request with the put method,
```

Comment:
"Retrieves http response for a request with the **_post_** method," - documentation needs to reflect http method